src/tests/functional/plugin/gna/scale_factors_tests/weighable_layer_without_fq.cpp

```diff
@@ -56,22 +56,23 @@ class WeighableLayerWithoutFqTest :
 
         auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
         auto params = ngraph::builder::makeParams(ngPrc, {inputShape});
+        auto relu = std::make_shared<ngraph::opset8::Relu>(params[0]);
         auto fq1 = std::make_shared<ngraph::opset8::FakeQuantize>(
-            params[0],
-            ngraph::opset8::Constant::create(ngraph::element::f32, {1}, {1.}),
-            ngraph::opset8::Constant::create(ngraph::element::f32, {1}, {1.}),
-            ngraph::opset8::Constant::create(ngraph::element::f32, {1}, {1.}),
-            ngraph::opset8::Constant::create(ngraph::element::f32, {1}, {1.}),
-            255);
+            relu,
+            ngraph::opset8::Constant::create(ngraph::element::f32, {1}, {-10.}),
+            ngraph::opset8::Constant::create(ngraph::element::f32, {1}, {10.}),
+            ngraph::opset8::Constant::create(ngraph::element::f32, {1}, {-10.}),
+            ngraph::opset8::Constant::create(ngraph::element::f32, {1}, {10.}),
+            static_cast<uint32_t>(std::numeric_limits<uint16_t>::max()) + 1);
         auto constant = ngraph::builder::makeConstant(ngPrc, constantShape, std::vector<float>{}, true);
         auto fq2 = std::make_shared<ngraph::opset8::FakeQuantize>(
             constant,
-            ngraph::opset8::Constant::create(ngraph::element::f32, {1}, {1}),
-            ngraph::opset8::Constant::create(ngraph::element::f32, {1}, {1.}),
-            ngraph::opset8::Constant::create(ngraph::element::f32, {1}, {1.}),
-            ngraph::opset8::Constant::create(ngraph::element::f32, {1}, {1.}),
-            255);
-        auto concat = ngraph::builder::makeConcat({fq1, fq2}, 0);
+            ngraph::opset8::Constant::create(ngraph::element::f32, {1}, {-10}),
+            ngraph::opset8::Constant::create(ngraph::element::f32, {1}, {10.}),
+            ngraph::opset8::Constant::create(ngraph::element::f32, {1}, {-10.}),
+            ngraph::opset8::Constant::create(ngraph::element::f32, {1}, {10.}),
+            static_cast<uint32_t>(std::numeric_limits<uint16_t>::max()) + 1);
+        auto concat = ngraph::builder::makeConcat({fq1, fq2}, 1);
         function = std::make_shared<ngraph::Function>(concat, params, "WeighableLayerWithoutFq");
```

Comment:
Could you please explain why it is named `WeighableLayerWithoutFq`? I don't understand why it is Weighable and why WithoutFq

Reply:
As far as I remember this test was written for issue with quantization of ConcatAlignFilter. This layer is weightable and it's inserted by GNA plugin, so POT doesn't quantize it. So, I think, it's a reason of this test name. There is a PR where this test was added: https://github.com/openvinotoolkit/openvino/pull/7834